docs/libcurl/opts/CURLOPT_WS_OPTIONS.md

```diff
@@ -44,6 +44,11 @@ callback.
 In raw mode, libcurl does not handle pings or any other frame for the
 application.
 
+## CURLWS_NOAUTOPONG (2)
+
+Disable the automatic reply to PING messages. This means users must
+send a PONG message with "curl_ws_send".
```

Comment:
```suggestion
send a PONG message with curl_ws_send(3).
```

makes it get "linked" correctly in manpages on the website.

Maybe also mentioned that this feature is added in 8.10.0 ?